app/src/main/java/services/authhandler/AuthHandler.java

```diff
@@ -0,0 +1,66 @@
+package services.authhandler;
+
+import android.util.Log;
+
+import com.amplifyframework.auth.AuthUserAttributeKey;
+import com.amplifyframework.auth.options.AuthSignUpOptions;
+import com.amplifyframework.core.Amplify;
+
+public class AuthHandler{
+
+    public static void signUp(String email, String username, String password, AuthEvents eventStatus){
+        AuthSignUpOptions options = AuthSignUpOptions.builder()
+                .userAttribute(AuthUserAttributeKey.email(), email)
+                .build();
+        Amplify.Auth.signUp(username, password, options,
+                result -> {
+                eventStatus.onSuccess();
+                Log.i("AuthQuickStart", "Result: " + result.toString());
```

Comment:
Extract "AuthQuickStart" to a `TAG` variable

Reply:
Done